### PR TITLE
Use utc date

### PR DESCRIPTION
**Pull request by LauraSManns**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 numpy
-pandas<=1.0.3
```
Comment (IvanKolesarEquinor) [inline requirements.txt:2]: This change should be in separate commit

File: tests/output/test_bazefetcher.py [hunk 2]
```diff
@@ -6,17 +6,27 @@
...
 
 eps = timedelta(microseconds=1)
 
+
+def assert_series_equal_wrap(_arg1, _arg2):
+    if sys.version_info[1] < 6:
+        pd.testing.assert_series_equal(_arg1, _arg2)
+    else: 
+        pd.testing.assert_series_equal(_arg1, _arg2, check_freq=False)
+
```
Comment (IvanKolesarEquinor) [inline tests/output/test_bazefetcher.py:21]: I saw this function in multiple places. It would be great if we could avoid code duplication. Can we move it to some separate module and import from it?

Wouldn't it be better to check if function has the parameter 'check_freq' instead of just general checking of python version?

**Review by IvanKolesarEquinor — CHANGES_REQUESTED** [review]
Also, please check formatting test from circleci:
https://app.circleci.com/pipelines/github/equinor/camille/55/workflows/3fdc2940-f28c-4246-a1d1-9f673353331a/jobs/364

There are several format issues, which should be addressed

**Review by JensGM — APPROVED** [review]
Looks very good!

File: camille/source/bazefetcher.py [hunk 3]
```diff
@@ -6,6 +6,8 @@
 import pandas as pd
 from collections import abc
 from math import ceil
+from ..util import utcdate
```
[the remark this answers is not in the document]
Comment (IvanKolesarEquinor) [reply camille/source/bazefetcher.py:9]: How about this one? camille.util does not work?

**Review by IvanKolesarEquinor — CHANGES_REQUESTED** [review]
One last change, can we squash (or more precise fixup) the last commit? I mean 31bf0098b275e462fa513b9e420778eccdd4d811 - "Fix one code formatting error".
